cmake/Modules/opm-upscaling-prereqs.cmake

```diff
@@ -27,5 +27,6 @@ set (opm-upscaling_DEPS
 	dune-grid REQUIRED;
 	opm-common REQUIRED;
 	opm-core REQUIRED;
+        opm-output REQUIRED;
 	dune-cornerpoint REQUIRED"
```

Comment:
Since the opposite also is true (dune-cornerpoint does not require opm-output) I do not think it needs to be changed. I'd say that the above is closer to a "canonical" ordering since output was just split out of core.